README.md

```diff
@@ -0,0 +1,54 @@
+# Testplane testing library
+
+## Introduction
+[Testing-library](https://testing-library.com/) is a collection of tools for testing web application user interfaces, focused on creating reliable and maintainable tests by emphasizing user behavior. The main advantage of `testing-library` is its focus on interaction with interface elements. And in testplane, you can use the element search methods provided by the `testing-library` itself.
+
+## Connection
```

Comment:
I think connection has a slightly different meaning, imo "installation" fits better here, but not critical to me

Reply:
Fixed